### ISSUE TITLE
Disable / opt-out of rel="preload" link tags in the head

### ISSUE THREAD

**Issue by arthurvi** [graysonhicks/gatsby-plugin-loadable-components-ssr#47]
The default behaviour is to add a `<link rel=preload>` tag for every chunk. 

```
// Set link rel="preload" tags in the head to start the request asap. This will NOT parse the assets fetched
setHeadComponents(extractor.getLinkElements()); // Set script and style tags at the end of the document to parse the assets.
```

We have the [flexible layout component use-case](https://www.gatsbyjs.com/blog/flexible-fine-grained-code-splitting-with-gatsby-loadable-components/). `gatsby-plugin-loadable-components-ssr` (or somewhere else?) seems to mark each component as high priority by setting the `<link rel=preload>` tag. Would you accept a PR to opt-out of this behaviour? Using Webpack comments then allows you to control it yourself (eg. via `/* webpackPreload: true */`).

By doing this we could mark only the hero as critical, and all the sections below the fold as less critical?

**Comment by graysonhicks** [comment]
Hi @arthurvi !  Yea I think that makes sense.  I'll have to mess around to see how fine-grain a user could control that.  I think disabling would have to come with a pretty big warning that you would then need inline comments.  I'll keep you updated!

**Comment by lilyntalmers** [comment]
Hey @graysonhicks! Thank you so much for making loadable SSR so much easy for GatsbyJS.😊

As @arthurvi pointed out, [webpack](https://webpack.js.org/guides/code-splitting/#prefetchingpreloading-modules) have a way to **preload the chunks** by `import(/* webpackPreload: true */ 'ChartingLibrary');`. But **gatsby-plugin-loadable-components-ssr** put `<link rel=preload>` for every chunk. Which preloads every chunk and doesn't give us the option to manually choose, thus downloading the chunks that are not instantly needed.😭

I have been waiting for a solution since January 2022. It will be really helpful if by default the plugin preloads every chunk using  `<link rel=preload>` but gives us a option to disable this behavior in plugin config.

This way people can keep using the plugin without changing anything but we also get the option to disable it for our requirements. I strongly believe that me and many others will highly appreciate it!😊

**Comment by graysonhicks** [comment]
Hi @lilyntalmers yes the preload tags are added [here](https://github.com/graysonhicks/gatsby-plugin-loadable-components-ssr/blob/master/src/gatsby-ssr.js#L17).  Could definitely add an option to disable that.  Will try to do that this week.

**Comment by lilyntalmers** [comment]
Thank you so much @graysonhicks for all the help. 😊

I’m looking forward to it! 🤩

**Comment by graysonhicks** [comment]
This should be available in `4.3.0` by passing in the option `preloadTags: false`:

```
  {
      resolve: "gatsby-plugin-loadable-components-ssr",
      options: {
         preloadTags: false
      },
  }
```

Thanks @lilyntalmers @arthurvi !

**Comment by lilyntalmers** [comment]
Thank you so much @graysonhicks !!! 😭
